### PR TITLE
Feature/storyboard instantiate

### PR DESCRIPTION
**Pull request by denisq1991**
Added a section on our storyboardInstatiate protocol as discussed in https://digitalcrew.teamwork.com/#tasks/12096245

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -553,6 +553,34 @@ When writing didSets they can be a few lines long if they are related to setting
...
+```swift
+  static var mainStoryboard: UIStoryboard {
+
+    return UIStoryboard("Tasks")
+  }
+```
+
+would point to Tasks.storyboard.
+
+Once you've set this value you must simply call `instantiateFromMainStoryboard()` on the ViewController and the you will be returned an instance of that ViewController.
```
Comment (dfreniche) [inline README.markdown]: "and then you" --> typo?

File: README.markdown [hunk 2]
```diff
@@ -553,6 +553,34 @@ When writing didSets they can be a few lines long if they are related to setting
...
+
+eg. `ProjectCollectionViewController.instantiateFromMainStoryboard()`
```
Comment (dfreniche) [inline README.markdown:583]: Where is `ProjectCollectionViewController.instantiateFromMainStoryboard` defined?

**Review by dfreniche — CHANGES_REQUESTED** [review]
Some doubts to be cleared

File: README.markdown (hunk 2 above)
Comment (rkreutz-teamwork) [reply README.markdown:583]: The protocol itself defines it. It's maaagic

Comment (dfreniche) [reply README.markdown:583]: OK, thought it was something from Apple, now I see it's a function inside the protocol 👍

**Review by josealobato — APPROVED** [review]

**Review by janodev — APPROVED** [review]

**Comment by janodev** [comment]
Approved, but there is a [typo](https://github.com/Teamwork/swift-style-guide/pull/18#discussion_r235443301).

**Comment by NSLeoFont** [comment]
Please remove me from here...Receiving mails...Btw....Nice to meet you guys! Hope everything is going fine...All the best! Leo

**Pull request merged**